python/eos/analysis.py

```diff
@@ -22,6 +22,9 @@
 import scipy
 import pypmc
 
+from eos import error, info, warn
+import dynesty
```

Comment:
Please have a look how we handle importing ``pypmc``. It is more graceful to import the module only when you know you need it, i.e., in the ``sample_nested`` method.